Source/WebInspectorUI/UserInterface/Views/FilterBar.js

```diff
@@ -38,9 +38,10 @@ WI.FilterBar = class FilterBar extends WI.Object
         this._inputField.type = "search";
         this._inputField.placeholder = WI.UIString("Filter");
         this._inputField.spellcheck = false;
-        this._inputField.incremental = true;
-        this._inputField.addEventListener("search", this._handleFilterChanged.bind(this));
-        this._inputField.addEventListener("input", this._handleFilterInputEvent.bind(this));
+        this._inputFieldInputThrottler = new Throttler(this._handleFilterInputEvent.bind(this), 250);
+        this._inputField.addEventListener("input", (event) => {
+            this._inputFieldInputThrottler.fire(event);
+        }, false);
```

Comment:
i think you actually have this backwards.  the old `_handleFilterChanged` is what should now be used to handle `"input"` instead of `_handleFilterInputEvent` (and we can probably entirely get rid of `_handleFilterInputEvent`)

i'd remove `_handleFilterInputEvent` entirely and then rename `_handleFilterChanged` to `_handleFilterInput` (since it's now for an `"input"` instead of `"search"`)
```
this._handleFilterInputThrottler = new Throttler(this._handleFilterInput.bind(this), 250);
this._inputField.addEventListener("input", (event) => {
    if (this._inputField.value)
        this._handleFilterInputThrottler.fire(event);
    else
        this._handleFilterInputThrottler.force(event);
});
```
note that we do the `force` in order to mimic the old behavior where clearing the `WI.FilterBar` would immediately trigger instead of on the delay from `"search"`

nit: we dont need the `, false` in this case

Reply:
Thank you for clarifying, added.